core/geometry/src/geometry3d/BarycentricTriangle.ts

```diff
@@ -560,7 +572,7 @@ export class BarycentricTriangle {
   public snapLocationToEdge(
     location: TriangleLocationDetail,
     distanceTolerance: number = Geometry.smallMetricDistance,
```

Comment:
I dont remember why I made this chage. I think you told me to but dont remember why!

Reply:
It's good. We want that tighter default snap tolerance to correspond to the snap tol in the new method.